src/routes/url-route.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+
+import { createUrl, deleteUrl, getUrl, getUrls, updateUrl } from '../controllers/url-controller';
+import { validatePostUrl, validateUpdateUrl } from '../middlewares/url-validator';
+
+const router = express.Router();
+
+router.post('/shorten', validatePostUrl, createUrl);
+router.get('/urls', getUrls);
+router.get('/url/:id', getUrl);
+router.put('/url/:id', validateUpdateUrl, updateUrl);
+router.delete('/url/:id', deleteUrl);
```

Comment:
According to the spec, this should be the path

```suggestion
router.get('/urls/:id', getUrl);
router.put('/urls/:id', validateUpdateUrl, updateUrl);
router.delete('/urls/:id', deleteUrl);
```

Reply:
done